### DISCUSSION TITLE
Server Error when login with wrong username and password (Netbox running in Azure Web App)

### DISCUSSION THREAD

**Discussion by Cobajar** [netbox-community/netbox#13660] [Help Wanted!]
I have deployed Netbox as Azure Web App (using Dockerhub as image source).

Everything seems to work fine so far, except when I login with wrong username / password.

I think netbox tries to report the failed login in the logs and uses the "x-forwarded-for" header (maybe?). Unfortunately, Azure adds the port to the ip address and netbox seems to have a problem with that.

![Netbox_Azure_FailedLogin](https://github.com/netbox-community/netbox/assets/48453066/9bc5a083-9635-4885-8c6a-ea29d1e7bafc)

A check should possibly be built into the login logic to see whether the source IP contains a ":", if so, split and only parse the IP address.

does anyone have any ideas?

Thank you,
Coby

**Comment by candlerb** [comment]
Could you get a full backtrace? Either set DEBUG=True temporarily in configuration.py, or set ADMINS and EMAIL so that all such exceptions get mailed out automatically.
